service/src/test/java/bio/terra/workspace/common/BaseAwsConnectedTest.java

```diff
@@ -23,7 +23,7 @@ public class BaseAwsConnectedTest extends BaseTest {
   @MockBean protected FeatureService featureService;
 
   @BeforeAll
-  public void setup() {
+  public void init() throws Exception {
```

Comment:
is the throws neede?

Reply:
Yes, the overloaded definition in subclass needs this